src/NJsonSchema/JsonSerializerSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NJsonSchema.CodeGeneration.CSharp
+{
+    public class JsonSerializerSettings
+    {
+        public Newtonsoft.Json.DateParseHandling? DateParseHandling { get; set; }
+
+        public string DateFormatString { get; set; }
+
+        override
+        public string ToString() {
+            var settingsString = DateParseHandling == null ? 
+                                    string.Empty : 
+                                    $"DateParseHandling = {DateParseHandling.ToString()}";
```

Comment:
I think the namespace and type is missing, eg. 

    $"DateParseHandling = Newtonsoft.Json.DateParseHandling.{DateParseHandling.ToString()}";

Reply:
It is referring to the Property DateParseHandling inside JsonSerializerSettings.